qa/rpc-tests/wallet_bench.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python3
```

Comment:
this file seems unrelated to this PR

Reply:
well, its a new QA "test" that benchmarks the wallet...